### ISSUE TITLE
Blinking when sliding under IE 11

### ISSUE THREAD

**Issue by turkovich** [jedrzejchalubek/Glide.js#154]
Slides are blinking under IE11 in carousel mode.

**Comment by jedrzejchalubek** [comment]
This could be caused by dozen of things. Probably some other element, which is animated too, affecting slider.

This issue is more addressed to "How browsers repaints view". I can't do much in here.

**Comment by turkovich** [comment]
The same problem is present on your site. Demonstration of problem https://gyazo.com/5888014d3f09c072bda026ba76bf4d23

Blinks but not slide.

**Comment by Nora0707** [comment]
Is there already a solution to this? Thanks a lot.

**Comment by mindhalt** [comment]
Any update?

**Comment by willans** [comment]
I thought I'd done something wrong. Checked the demo, nope that does the blinking instead of sliding. Any update?

**Comment by willans** [comment]
Just an update, seems fine in BrowserStack. However, the bug about it disappearing was on a SurfaceBook (with a touch screen) if that's any help.

**Comment by HauntedSmores** [comment]
If it helps, Im noticing that programmatic navigation works smoothly, its autoplay that is flashing instead of sliding.

**Comment by fraser-sideas** [comment]
Seem to have found a workaround relating to style attribute `backface-visibility: hidden` under the glide__slides class. If you set it to visible then it appears to fix IE issue and doesn't have adverse effects on other browsers, in our application anyway.

For example
 ` let slides = document.getElementsByClassName("glide__slides")
   for (var i=0; i<slides.length; i++){
        slides[i].style.backfaceVisibility = "visible";
   }`

**Comment by corford** [comment]
Just wanted to add a "me too". IE11 blinks/flashes rather than scrolls (as can be seen on the official glidejs demos themselves). The fix suggested by @fraser-sideas (setting backface-visiblity to "visible") corrected the problem (with no apparent impact to other browsers).

**Comment by mindhalt** [comment]
Yeah, fix is good, but better was to set in CSS instead of JS-way.

**Comment by ShoShindo** [comment]
Is this all right?
`
.glide__slides {
  backface-visibility: visible;
}
`

**Comment by maldo0548** [comment]
@ShoShindo That's what I did and it worked for me! Although I went with the id selector, #glide__slides to give it more specificity.
